project/Dependencies.scala

```diff
@@ -0,0 +1,11 @@
+import sbt._
+
+object Dependencies {
+  lazy val scalaTest = "org.scalatest" %% "scalatest" % "3.1.0"
```

Comment:
Can we use codacy-sbt?

Reply:
Codacy sbt does not have the most recent version. But yes we can

Reply:
can we maybe then have a recent one and the legacy one in codacySbt?